draft-ietf-quic-recovery.md

```diff
@@ -1274,7 +1277,8 @@ GetPtoTimeAndSpace():
   duration = (smoothed_rtt + max(4 * rttvar, kGranularity))
       * (2 ^ pto_count)
   // Arm PTO from now when there are no inflight packets.
-  if (no in-flight packets):
+  // bytes_in_flight is defined in the next appendix.
+  if (bytes_in_flight <= 0):
```

Comment:
How can bytes_in_flight be negative?

Reply:
`bytes_in_flight` is non negative. But this coding style follows the traditional practice. ;)

Reply:
bytes_in_flight isn't introduced until the congestion control section, which is why I didn't do this.  We could merge the pseudocode from both, but I believe doing that is a larger project than this PR, so I don't want to do this in this PR.